app/src/main/res/layout/state_fragment.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Since there was no flow created yet and I was directly working on Content card I am bypassing state fragment and from ExplorationActivity it is directly redirecting to ContentListFragment. Will be including placeholders accordingly while working on the flow

Reply:
Ah, will those be added in this PR?